src/decoder/lattice-faster-online-decoder.h

```diff
@@ -335,14 +336,25 @@ class LatticeFasterOnlineDecoder {
   BaseFloat GetCutoff(Elem *list_head, size_t *tok_count,
                       BaseFloat *adaptive_beam, Elem **best_elem);
 
-  /// Processes emitting arcs for one frame.  Propagates from prev_toks_ to cur_toks_.
-  /// Returns the cost cutoff for subsequent ProcessNonemitting() to use.
-  BaseFloat ProcessEmitting(DecodableInterface *decodable);
+  // Processes emitting arcs for one frame.  Propagates from prev_toks_ to cur_toks_.
+  // Returns the cost cutoff for subsequent ProcessNonemitting() to use.
+  // To reduce vtable lookup cost in openfst based graph search, 
+  // make ProcessEmitting() and ProcessNonemitting() be templated on the FstType,
+  // and create wrapper functions, ProcessEmittingWrapper(), which will check 
+  // the actual FstType; if it's ConstFst or VectorFst, it will call 
```

Comment:
fix this comment.

Reply:
@chenzhehuai, did you see this?
I asked you to fix this comment many times, by github comment and by email.  For your convenience I'll repeat here an email I sent you separately, which you also seem to have missed.  It's a very small point, but after asking 4 or 5 times, it starts to get annoying.
```
thanks.
Regarding the comment: please change:
// Processes emitting arcs for one frame.  Propagates from prev_toks_ to cur_toks_.
   // Returns the cost cutoff for subsequent ProcessNonemitting() to use.
  // To reduce vtable lookup cost in openfst based graph search, 
   // make ProcessEmitting() and ProcessNonemitting() be templated on the FstType,
    // and create wrapper functions, ProcessEmittingWrapper(), which will check 
    // the actual FstType; if it's ConstFst or VectorFst, it will call 
    // ProcessEmitting() with one of those two types explicitly; otherwise it 
  // will call it with the regular Fst interface (to cover the other cases).
   // will call it with the regular Fst interface .

to:

   /// Processes emitting arcs for one frame.  Propagates from prev_toks_ to cur_toks_.
   /// Returns the cost cutoff for subsequent ProcessNonemitting() to use.
   /// Templated on FST type for speed; called via ProcessEmittingWrapper().


that's what I was trying to say before.
Be careful: this is in 2 places.
```